source/_integrations/generic.markdown

```diff
@@ -15,18 +15,13 @@ The `generic` camera platform allows you to integrate any IP camera or other URL
 
 Home Assistant will serve the images via its server, making it possible to view your IP cameras while outside of your network. The endpoint is `/api/camera_proxy/camera.[name]`.
 
-## Configuration
+{% include integrations/config_flow.md %}
 
-To enable this camera in your installation, add the following to your `configuration.yaml` file:
+You must enter a URL in at least one of the fields **Still Image URL** or **Stream Source URL**, the others are optional.
 
-```yaml
-# Example configuration.yaml entry
-camera:
-  - platform: generic
-    still_image_url: http://194.218.96.92/jpg/image.jpg
-```
+[Templates](/topics/templating/) are allowed in the URL fields, which can be used to select different images or parameterize the URL depending on the status of sensors.  Template validity and network access are checked during the configuration steps.
 
-{% configuration %}
+{% configuration_basic %}
 still_image_url:
   description: "The URL your camera serves the image on, e.g., `http://192.168.1.21:2112/`. Can be a [template](/topics/templating/). At least one of still_image_url or stream_source must be provided."
   required: false
```

Comment:
`required`, `type`, `default` and many more, are not part of `configuration_basic`, please clean it up

Reply:
I swapped it from configuration to configuration_basic to try to remove the 'help finding your YAML file' automatic section. Is there another way to turn this off?

Reply:
`configuration_basic` only has field names with the `description` key. All others do not apply and can be removed.

Reply:
Ok, done!

Reply:
Ok, done.